vignettes/third-party-integrations.Rmd

```diff
@@ -34,7 +34,7 @@ styler functionality is available in other tools, most notably
 
 -   in the *format-all* command for Emacs in [emacs-format-all-the-code](https://github.com/lassik/emacs-format-all-the-code).
 
--   As a [Jupyterlab code formatter](https://jupyterlab-code-formatter.readthedocs.io/en/latest/installation.html#r-code-formatters).
+<!-- FIXME -   As a [Jupyterlab code formatter](https://jupyterlab-code-formatter.readthedocs.io/en/latest/installation.html#r-code-formatters). -->
```

Comment:
I can't find the correct link for this, if one even exists.

If not, I can just remove it.

Reply:
https://ryantam626.github.io/jupyterlab_code_formatter/index.html#